src/com/georgidinov/Main.java

```diff
@@ -0,0 +1,9 @@
+package com.georgidinov;
+
+public class Main {
```

Comment:
Javadoc, unit tests, at the demo don't keep code which is not used by main functionality here, those should be on the test side. tests.